news/2025/2025-06-01-mgts-2025-registrations.md

```diff
@@ -0,0 +1,106 @@
+
+### Songs
+
+This wouldn't be a GTS event without entirely original songs and beatmaps to accompany it, only this time of course *you* will be collaborating for the mapping part!
+
+Check them all out below:
+
+![](/wiki/shared/news/2025-06-01-mgts-2025-registrations/illusoryprogram97.png)
+
+| Song | [Kyotsugyon](https://soundcloud.com/kyotsugyon) & [Sh0wtime](https://soundcloud.com/sh0wtimelol) - [ILLUSORYPROGRAM97](https://soundcloud.com/kyotsugyon/illusoryprogram97) |
+| :-- | :-- |
+| **Length** | 5:10 |
+| **BPM** | 149–198 (198) |
+| **Note** | Tiebreaker style |
+
+<audio controls>
+    <source src="https://assets.ppy.sh/media/news/mgts-2025-illusoryprogram97.mp3">
+</audio>
+
+![](/wiki/shared/news/2025-06-01-mgts-2025-registrations/axisbreak.jpg)
+
+| Song | [X](https://soundcloud.com/foodbot)/[Y](https://soundcloud.com/mrcool909090)/[Z](https://soundcloud.com/silis) - axis::BREAK |
+| :-- | :-- |
+| **Length** | 2:52 |
+| **BPM** | 202 |
+| **Note** | Consistency pick |
+
+<audio controls>
+    <source src="https://assets.ppy.sh/media/news/mgts-2025-axisbreak.mp3">
+</audio>
+
+![](/wiki/shared/news/2025-06-01-mgts-2025-registrations/hatching.png)
+
+| Song | [Faxyne](https://soundcloud.com/faxyne) vs. [blobdash](https://soundcloud.com/blobdash) - [Hatching](https://blobdash.bandcamp.com/track/hatching) |
+| :-- | :-- |
+| **Length** | 2:38 |
+| **BPM** | 200 |
+| **Note** | Tech pick |
+
+<audio controls>
+    <source src="https://assets.ppy.sh/media/news/mgts-2025-hatching.mp3">
+</audio>
+
+## Rules
+
+Everything about the contest's rules, procedures, and judging criteria can be found in both the [contest's website](https://gtsosu.com/2025/mgts/home) and its [osu! wiki page](/wiki/Contests/MGTS/2025) — please make sure you and your team go through them thoroughly in order to make sure you end up with a valid submission!
+
+## How do I participate?
+
+First of all, joining the [Discord server](https://discord.com/invite/3mGC3HB) is mandatory for participation, because all important announcements will be communicated there.
+
+Your next step will be to form a team with 2 to 3 members, and have the captain [register the team in the GTS website](https://gtsosu.com/2025/mgts/home).
+
+Once that's done, download one of the two beatmap templates below to start mapping:
+
+- [Beatmap template 1: ILLUSORYPROGRAM97 + axis\:\:BREAK](https://assets.ppy.sh/media/news/MGTS%202025%20Template%201.osz)
+- [Beatmap template 2: ILLUSORYPROGRAM97 + Hatching](https://assets.ppy.sh/media/news/MGTS%202025%20Template%202.osz)
+
+Make sure to update the beatmap creator's name to **your team's captain's name** in order to have a valid submission.
+
+After you're done mapping, have **your team's captain** head over to the [submission page](https://osu.ppy.sh/community/contests/248) to submit your entry. However, don't be too slow about it because you have **29 days** before the submission deadline!
```

Comment:
```suggestion
After you're done mapping, have **your team captain** head over to the [submission page](https://osu.ppy.sh/community/contests/248) to submit your entry. However, don't be too slow about it because you have **29 days** before the submission deadline!
```